keras_cv/utils/test_utils.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+
+import tensorflow as tf
+
+from keras_cv import core
+from keras_cv import layers as cv_layers
+
+
+def exhaustive_compare(obj1, obj2):
+    classes_supporting_get_config = (
+        core.FactorSampler,
+        tf.keras.layers.Layer,
+        cv_layers.BaseImageAugmentationLayer,
```

Comment:
Is this redundant, since it subclasses Layer?

Id it's not redundant, and we need it, do we also need to list `Loss` here?